### PR TITLE
[Indigo][moveit] Add/update metapackages.

### PR DESCRIPTION
**Pull request by 130s**
- Moving existing metapackages from an individual repo to `moveit` repo (for Indigo).
- Adding a new metapackage (IJK).

**Question**: in both cases above, `bloom` has not yet been run (ie. release repo not updated). I'm afraid this PR would fail without those packages being bloomed? (If so I think I'll have to close this).

(Changes here is discussed in https://github.com/ros-planning/moveit_metapackages/issues/11)

### CODE REVIEW

**Comment by 130s** [comment]
Closing. I assume moving packages to another repo still requires release artifact generated in the new release repo and that's why [package.xml is not found](https://github.com/ros-planning/moveit/blob/be24a76ae73a3290f4d5681d727b93b1b1167747/moveit_runtime/package.xml) for the moved packages. I'll bloom shortly.

**Comment by 130s** [comment]
tldr; Please review. The change is for indigo only.

The packages listed in the change are already moved to a new devel repository, but `bloom` fails because they are already listed in rosdistro within the old repo. Therefore, in order to complete the move to a new devel repo, they need to be removed from rosdistro. Then `bloom` needs to be run shortly to keep the period where those packages in transition are not available as short as possible.

I removed the change from J and K, which can be done by bloom only (as already done for K https://github.com/ros/rosdistro/pull/13809).

**Comment by dirk-thomas** [comment]
What should be reviewed here? The ticket has been closed.

**Comment by wjwwood** [comment]
@130s this pr is closed?

**Comment by 130s** [comment]
Reopen button is disabled so that I can't reopen. Maybe I'll just open a new PR instead of digging further for now.
